tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -226,3 +226,117 @@ def test_cpu_template(test_microvm_with_ssh, network_config, cpu_template):
+                          "CMPXCHG16B instruction": "true",
+                          "process context identifiers": "true",
+                          "SSE4.1 extensions": "true",
+                          "SSE4.2 extensions": "true",
+                          "extended xAPIC support": "true",
+                          "POPCNT instruction": "true",
+                          "time stamp counter deadline": "true",
+                          "XSAVE/XSTOR states": "true",
+                          "OS-enabled XSAVE/XSTOR": "true",
+                          "AVX: advanced vector extensions": "true",
+                          "F16C half-precision convert instruction": "true",
+                          "RDRAND instruction": "true",
+                          "hypervisor guest status": "true",
+                          }
+    thermal_and_power_mgmt = {"ARAT always running APIC timer": "true"}
+
+    extended_features = {"FSGSBASE instructions": "true",
+                         "IA32_TSC_ADJUST MSR supported": "true",
+                         "SMEP supervisor mode exec protection": "true",
+                         "enhanced REP MOVSB/STOSB": "true",
+                         "SMAP: supervisor mode access prevention": "true"}
+
+    xsave_0xd_0 = {"XCR0 supported: x87 state": "true",
+                   "XCR0 supported: SSE state": "true",
+                   "XCR0 supported: AVX state": "true"}
+
+    xsave_0xd_1 = {"XSAVEOPT instruction": "true"}
+
+    extended_080000001_edx = {"SYSCALL and SYSRET instructions": "true",
+                              "64-bit extensions technology available": "true",
+                              "execution disable": "true", "RDTSCP": "true"}
+
+    intel_080000001_ecx = {"LAHF/SAHF supported in 64-bit mode": "true"}
+    adv_pwr_mgmt = {"TscInvariant": "true"}
+
+    utils.check_guest_cpuid_output(test_microvm, "cpuid -1", None, '=',
+                                   feature_info_1_edx)
+    utils.check_guest_cpuid_output(test_microvm, "cpuid -1", None, '=',
+                                   feature_info_1_ecx)
+    utils.check_guest_cpuid_output(test_microvm, "cpuid -1", None, '=',
+                                   thermal_and_power_mgmt)
+    utils.check_guest_cpuid_output(test_microvm, "cpuid -1", None, '=',
+                                   extended_features)
+    utils.check_guest_cpuid_output(test_microvm, "cpuid -1", None, '=',
+                                   xsave_0xd_0)
+    utils.check_guest_cpuid_output(test_microvm, "cpuid -1", None, '=',
+                                   xsave_0xd_1)
+    utils.check_guest_cpuid_output(test_microvm, "cpuid -1", None, '=',
+                                   extended_080000001_edx)
+    utils.check_guest_cpuid_output(test_microvm, "cpuid -1", None, '=',
+                                   intel_080000001_ecx)
+    utils.check_guest_cpuid_output(test_microvm, "cpuid -1", None, '=',
+                                   adv_pwr_mgmt)
```

Comment:
Can we have all the features above in one single giant dictionary, and add the leafs only as comments. For example:
```
common_enabled_features = {
    ...
    #intel_080000001_ecx
    "LAHF/SAHF supported in 64-bit mode": "true",
    #adv_pwr_mgmt
    "TscInvariant": "true"
}
```

And then call `check_guest_cpuid_output()` only once.

Reply:
done.